x-pack/plugins/cloud_security_posture/public/components/take_action.tsx

```diff
@@ -255,17 +258,20 @@ const CreateDetectionRule = ({
   notifications,
   http,
   queryClient,
+  isRuleMuted = true,
 }: {
   createRuleFn: (http: HttpSetup) => Promise<RuleResponse>;
   setIsLoading: (isLoading: boolean) => void;
   closePopover: () => void;
   notifications: NotificationsStart;
   http: HttpSetup;
   queryClient: QueryClient;
+  isRuleMuted: boolean;
 }) => {
   return (
     <EuiContextMenuItem
       key="createRule"
+      disabled={isRuleMuted}
```

Comment:
Neat: UX Improvement suggestion:

Currently, there's no indication of why the option is disabled:

![image](https://github.com/elastic/kibana/assets/19270322/714a8366-c9da-41da-be68-27a02ecaefee)

It would be good for the user to add some tooltip information on the hover, so the user knows that this option is disabled because the benchmark rule is muted, similar as displayed today in the switch component

![image](https://github.com/elastic/kibana/assets/19270322/a6e67516-b76d-4602-90de-8f013498b06e)

Reply:
hmmm not sure what the correct message to put there, I can bring this up to Einat and add this as a Quick wins